src/contracts/middleware/BLSPubkeyRegistry.sol

```diff
@@ -91,6 +91,22 @@ contract BLSPubkeyRegistry is IBLSPubkeyRegistry, Test {
         return pubkeyHash;
     }
 
+    /// @notice Returns the index of the quorumApk index at `blockNumber` for the provided `quorumNumber`
```

Comment:
this comment is written like it's for a single entry but this actually is for (and returns) an array

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/81/commits/d4daf8507a5799729702562ebdd1e37b6f88caee